fieldmanager.php

```diff
@@ -358,33 +368,50 @@ function fm_calculate_context() {
+							// phpcs:ignore WordPress.Security.NonceVerification.Missing -- baseline
+							$calculated_context = array( 'term', sanitize_text_field( wp_unslash( $_POST['taxonomy'] ) ) );
 						}
-					} elseif ( ! empty( $_GET['action'] ) && 'fm_quickedit_render' === $_GET['action'] ) { // WPCS: input var okay.
+					// phpcs:ignore WordPress.Security.NonceVerification.Recommended -- baseline
+					} elseif ( ! empty( $_GET['action'] ) && 'fm_quickedit_render' === $_GET['action'] ) {
 						// Context = "quickedit".
-						$calculated_context = array( 'quickedit', sanitize_text_field( wp_unslash( $_GET['post_type'] ) ) ); // WPCS: input var okay.
-					} elseif ( isset( $_POST['action'] ) && 'heartbeat' === $_POST['action'] && isset( $_POST['data']['fm_context'] ) ) { // WPCS: CSRF ok.
+						// phpcs:ignore WordPress.Security.NonceVerification.Recommended -- baseline
+						$calculated_context = array( 'quickedit', sanitize_text_field( wp_unslash( $_GET['post_type'] ) ) );
+					// phpcs:ignore WordPress.Security.NonceVerification.Missing -- baseline
+					} elseif ( isset( $_POST['action'] ) && 'heartbeat' === $_POST['action'] && isset( $_POST['data']['fm_context'] ) ) {
 						$calculated_context = array(
-							sanitize_text_field( wp_unslash( $_POST['data']['fm_context'] ) ), // WPCS: CSRF ok.
-							isset( $_POST['data']['fm_subcontext'] ) ? sanitize_text_field( wp_unslash( $_POST['data']['fm_subcontext'] ) ) : null, // WPCS: CSRF ok.
+							// phpcs:ignore WordPress.Security.NonceVerification.Missing -- baseline
+							sanitize_text_field( wp_unslash( $_POST['data']['fm_context'] ) ),
+							// phpcs:ignore WordPress.Security.NonceVerification.Missing -- baseline
+							isset( $_POST['data']['fm_subcontext'] ) ? sanitize_text_field( wp_unslash( $_POST['data']['fm_subcontext'] ) ) : null,
 						);
 					}
 					break;
 				// Context = "term".
 				case 'edit-tags.php':
-				case 'term.php': // As of 4.5-alpha; see https://core.trac.wordpress.org/changeset/36308
-					if ( ! empty( $_POST['taxonomy'] ) ) { // WPCS: input var okay. CSRF ok.
-						$calculated_context = array( 'term', sanitize_text_field( wp_unslash( $_POST['taxonomy'] ) ) ); // WPCS: input var okay. CSRF ok.
-					} elseif ( ! empty( $_GET['taxonomy'] ) ) { // WPCS: input var okay.
-						$calculated_context = array( 'term', sanitize_text_field( wp_unslash( $_GET['taxonomy'] ) ) ); // WPCS: input var okay.
+				case 'term.php':
+					// phpcs:ignore WordPress.Security.NonceVerification.Missing -- baseline
+					if ( ! empty( $_POST['taxonomy'] ) ) {
+						// phpcs:ignore WordPress.Security.NonceVerification.Missing -- baseline
+						$calculated_context = array( 'term', sanitize_text_field( wp_unslash( $_POST['taxonomy'] ) ) );
+					// phpcs:ignore WordPress.Security.NonceVerification.Recommended -- baseline
+					} elseif ( ! empty( $_GET['taxonomy'] ) ) {
+						// phpcs:ignore WordPress.Security.NonceVerification.Recommended -- baseline
```

Comment:
I guess it wouldn't work with the baseliner in the sense that the baseliner wouldn't pick it up on a future run. It would also exempt new code in this block from following the newer rules. But I'm open to it if it seems like it's too much.

Reply:
> I guess it wouldn't work with the baseliner in the sense that the baseliner wouldn't pick it up on a future run.

This is from memory — we could test to be sure.

Reply:
No big deal either way 😄